### PR TITLE
added search function

### PR DESCRIPTION
**Pull request by NivoDev**

### CODE REVIEW

File: niv-rozanowich/tvmaze/shows.js [hunk 1]
```diff
@@ -0,0 +1,307 @@
+export const shows = [
```
Comment (davidmeirlevy) [inline niv-rozanowich/tvmaze/shows.js]: you could use the same shows file from the root directory.

Comment (davidmeirlevy) [reply niv-rozanowich/tvmaze/shows.js]: You can also remove this file..

Comment (NivoDev) [reply niv-rozanowich/tvmaze/shows.js]: Im not sure how to remove files

Comment (davidmeirlevy) [reply niv-rozanowich/tvmaze/shows.js]: Remove the file, then type git status for instructions.

**Pull request merged**